### ISSUE TITLE
RadPhi2 : what is trained on the Out of Chest X-Ray dataset ?

### ISSUE THREAD

**Issue by rkassana** [Stanford-AIMI/GREEN#7]
Hello, I was wondering if this model https://huggingface.co/StanfordAIMI/GREEN-RadPhi2 was trained on the Out-of Chest X-ray dataset as explained in Table 7 of the paper ?

Thank you !

**Comment by SophieOstmeier** [comment]
Hi, we only trained a RadLlama2 on Out-of Chest X-ray data at this point. We have not uploaded that model yet. I can do that soon. We also hope to be able to release the entire data set via physionet.

**Comment by SophieOstmeier** [comment]
Hi, I just uploaded the model that was trained on both Chest Xray data and Out of Chest X-Ray. This model was used in the paper. Note that this model was trained based on the GREEN-Radllama with the additional Out of Chest X-Ray dataset. 

https://huggingface.co/StanfordAIMI/GREEN-MM-RadLLaMA-7b

Hope that helps!
Sophie

**Comment by bhosalems** [comment]
But the link is dead.

**Comment by SophieOstmeier** [comment]
Can you now try again please?

**Comment by bhosalems** [comment]
yes. it is accessible now. thanks. I was wondering what is the typical green score you get? I see there were tables for deltas and correlation but what is typically considered a good value or is there any benchmark table that i missed in the paper?

**Comment by SophieOstmeier** [comment]
Yes for this question, table 8 is most relevant. The deltas tell us how good the performance of the fine-tuned model is against the GPT. The main contribution of this Section 5 in the paper is to show that this method including a more general dataset generation process is also adaptable for any type of radiology report if needed. For a comparison against radiologists, it would be great if someone publishes a dataset for that!

**Comment by bhosalems** [comment]
Ok, good to know this. But my question was more generic about the typical score values you get not specifically for "Out-of-Chest-X-Ray" data. I read that the range is 0 to 1 but what is typical green score that you observed in your experiments for the best model.

**Comment by SophieOstmeier** [comment]
We did not write that down in this case, because the score is always on the same scale. But let me see if we can make the data public and then it is easy to compute with `compute_green` in green_score/green.py
